recipes/hyperscan/all/conanfile.py

```diff
@@ -95,5 +105,35 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
-        self.cpp_info.names["pkg_config"] = "libhs"
-        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.names["cmake_find_package"] = "hyperscan"
+        self.cpp_info.names["cmake_find_package_multi"] = "hyperscan"
+
+        self.cpp_info.components["hs"].libs = ["hs"]
+        self.cpp_info.components["hs"].requires = ["boost::headers"]
+        self.cpp_info.components["hs"].names["cmake_find_package"] = "hs"
+        self.cpp_info.components["hs"].names["cmake_find_package_multi"] = "hs"
+        self.cpp_info.components["hs"].set_property("pkg_config_name", "libhs")
+
+
+        self.cpp_info.components["hs_runtime"].libs = ["hs_runtime"]
+        self.cpp_info.components["hs_runtime"].names["cmake_find_package"] = "hs_runtime"
```

Comment:
no pkg config for this one?

Reply:
Upstream doesn't provide a pkg config for this one, but I can add it in conanfile.